pkgs/applications/video/pitivi/default.nix

```diff
@@ -127,7 +127,7 @@ in python3Packages.buildPythonApplication rec {
     gst-plugins-bad
     gst-plugins-ugly
     gst-libav
-    gst-validate
+    gst-devtools
   ]);
```

Comment:
gst-transcoder is now part of gstreamer so it will likely conflict. There are also some upstream patches that might be relevant. Or maybe update pitivi.

Reply:
It hasn't conflicted so far, but independent of it, I cannot start `pitivi`: It fails with the probably unrelated error:

```
Traceback (most recent call last):
  File "/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/bin/..pitivi-wrapped-wrapped", line 169, in <module>
    _run_pitivi()
  File "/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/bin/..pitivi-wrapped-wrapped", line 146, in _run_pitivi
    from pitivi import application
  File "/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/lib/pitivi/python/pitivi/application.py", line 34, in <module>
    from pitivi.mainwindow import MainWindow
  File "/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/lib/pitivi/python/pitivi/mainwindow.py", line 42, in <module>
    from pitivi.medialibrary import AssetThumbnail
  File "/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/lib/pitivi/python/pitivi/medialibrary.py", line 173, in <module>
    class AssetThumbnail(Loggable):
  File "/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/lib/pitivi/python/pitivi/medialibrary.py", line 189, in AssetThumbnail
    EMBLEMS[status].append(GdkPixbuf.Pixbuf.new_from_file_at_size(
gi.repository.GLib.Error: gdk-pixbuf-error-quark: Couldn’t recognize the image file format for file “/nix/store/0qk27xn3za106wpg4l3miiqgfqhdk9s4-pitivi-0.999/share/pitivi/pixmaps/asset-proxied.svg” (3)
```

Reply:
Sounds like pitivi needs `strictDeps = false;`. I thought I already added that :woman_shrugging:

Reply:
@jtojnar I also just figured that out, doing it in PR #101398 (and I rebased this PR on top of that commit).